src/nfa/operations.cc

```diff
@@ -58,6 +66,9 @@ namespace {
         return LTSforSimulation.compute_simulation();
     }
 
+    //Something shorter? like simulation_quotient?
```

Comment:
Then I would suggest `compute_simulation_quotient()`.

Reply:
`simulation_reduction`?

Reply:
if we name functions by the returned thing, then quotient. reduction would be the name of the process used for getting the result.

Reply:
simulation and the simulation quotient are two different things no?

Reply:
yes sure, simulation is the relation, the quotient is the automaton you get by merging equivalent states